ga4gh/datamodel/reads.py

```diff
@@ -586,6 +622,8 @@ class SimulatedReadGroup(AbstractReadGroup):
     def __init__(self, parentContainer, localId, randomSeed, numAlignments=2):
         super(SimulatedReadGroup, self).__init__(parentContainer, localId)
         self._randomSeed = randomSeed
+        self._numAlignedReads = self._parentContainer.getNumAlignedReads()
+        self._numUnalignedReads = 0
```

Comment:
I think it's for a non-nullable field, so we have to put something in it. 0 seems like the best value in this situation.